packages/wonder-blocks-dropdown/components/single-select.js

```diff
@@ -50,9 +49,14 @@ type Props = {|
     light: boolean,
 
     /**
```

Comment:
I'm mixed on the additional style props, but I'm struggling to think of a better option, so I think this is fine!